### PR TITLE
8 create rss parser

### PR DESCRIPTION
**Pull request by bartoszboruta**

### CODE REVIEW

File: GPP-1/MainWindow.xaml.cs [hunk 1]
```diff
@@ -23,6 +10,7 @@ public partial class MainWindow : Window
         public MainWindow()
         {
             InitializeComponent();
+            RssParser.ParseNews("http://www.money.pl/rss/gielda.xml");
```
Comment (mateuszwu) [inline GPP-1/MainWindow.xaml.cs]: remove this

File: GPP-1/RssParser.cs [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+namespace GPP_1
+{
+    public static class RssParser
+    {
+        public static List<News> ParseNews(string url)
+        {
+            List<News> parsedNews = NewsFactory.CreateEmptyList();
+            var news = FetchRssData(url);
+            //TODO category regex
```
Comment (mateuszwu) [inline GPP-1/RssParser.cs]: remove it

File: GPP-1/RssParser.cs [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+using System.Windows;
+using System.Text.RegularExpressions;
+
+namespace GPP_1
+{
+    public static class RssParser
+    {
+        public static List<News> ParseNews(string url)
+        {
+            List<News> parsedNews = NewsFactory.CreateEmptyList();
```
Comment (mateuszwu) [inline GPP-1/RssParser.cs]: use `var` instead of `List<News>`

File: GPP-1/RssParser.cs [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+            foreach (var item in news.Items)
+            {
+                parsedNews.Add(ParsedNews(item));
+            }
+            return parsedNews;
+        }
+        
+        private static News ParsedNews(SyndicationItem news)
+        {
+            String category = "TODO";
```
Comment (mateuszwu) [inline GPP-1/RssParser.cs]: `var` instead of `String`

File: GPP-1/RssParser.cs [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+            MessageBox.Show(news.BaseUri.AbsoluteUri.ToString());
```
Comment (mateuszwu) [inline GPP-1/RssParser.cs]: remove it

File: GPP-1/RssParser.cs [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+            return parsedNews;
+        }
+        
+        private static News ParsedNews(SyndicationItem news)
+        {
+            String category = "TODO";
+
+            return NewsFactory.Create(
+                category.ToString(),
+                news.Summary.Text.ToString(),
```
Comment (PDendys) [inline GPP-1/RssParser.cs]: Item description retrieved with html markups

**Pull request merged**